ocaml/typing/typeclass.ml

```diff
@@ -1486,6 +1489,7 @@ let var_option =
 let rec approx_declaration cl =
   match cl.pcl_desc with
     Pcl_fun (l, _, _, cl) ->
+      let l = transl_label l in
```

Comment:
Similar problems as elsewhere. This is used when making a recursive class definition.